### ISSUE TITLE
Need a way to navigate quickly in images 

### ISSUE THREAD

**Issue by xvolks** [ryouchinsa/Rectlabel-support#15]
Our corpus of train documents is growing quickly (yesterday I had 320 images, today I have 2396 images).

We need to seek at random position (to take a template of boxes for example) in the flow of images and with Command + arrow it is just not possible.
A search by index and by partial/whole filename would be great.

For example:

- jump to (command + j), type _1568_ : RectLabel displays image # 1568
- search (command + f),  type _age*568_: RectLabel displays image named "MyGreatImage_1568.png" if it's the only match or a clickable list of all matches with a preview of each image (limited to 50 to avoid memory issue)

**Comment by ryouchinsa** [comment]
Thanks for reporting the problem.
We added the short cut key "command + J" to jump to the specified image index.

![2017-08-10 11 24 58](https://user-images.githubusercontent.com/1954306/29152204-15a16a98-7dc1-11e7-9f2d-cbf79add8ab1.png)

![2017-08-10 11 31 05](https://user-images.githubusercontent.com/1954306/29152213-290a84d4-7dc1-11e7-92db-0851ce5ba62c.png)

We submitted the new update to Apple.
When the new update is released, we will let you know.
Thank you.
